From 367d90bbb5bb4cd3f44da1108208a8eb3d913a0b Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Fri, 9 Sep 2022 16:44:51 +0300
Subject: [PATCH 10/30] s32cc: Make DFS base addresses available for MMU

Make DFS base addresses available for MMU settings
through headers.

Issue: ALB-9287
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 plat/nxp/s32/include/s32_clocks.h | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/plat/nxp/s32/include/s32_clocks.h b/plat/nxp/s32/include/s32_clocks.h
index 6495af3b0..e86a0608d 100644
--- a/plat/nxp/s32/include/s32_clocks.h
+++ b/plat/nxp/s32/include/s32_clocks.h
@@ -72,10 +72,16 @@ enum s32_dfs_type {
 	S32_DFS_NR	/* sentinel */
 };
 
-#define S32_DFS_BASE_ADDR	0x40054000ul
+#define S32_DFS_BASE_ADDR		(0x40054000ul)
+
 /* @dfs - One of the enum s32_dfs_type values */
 #define S32_DFS_ADDR(dfs)	(S32_DFS_BASE_ADDR + (dfs) * 0x4000)
 
+#define ARM_DFS_BASE_ADDR	S32_DFS_ADDR(S32_CORE_DFS)
+#define ARM_DFS_SIZE		(0x34)
+#define PERIPH_DFS_BASE_ADDR	S32_DFS_ADDR(S32_PERIPH_DFS)
+#define PERIPH_DFS_SIZE		(0x34)
+
 /* DFS register offsets */
 #define DFS_PORTRESET_OFF	0X14ul
 #define DFS_PORTRESET(dfs)	((S32_DFS_ADDR(dfs)) + DFS_PORTRESET_OFF)
-- 
2.17.1

